### PR TITLE
Closes #3167: Add `normal` to random number generators

### LINKED ISSUE #3167: Add `normal` to random number generators
Add `normal` to our generators, we already have a `fillRandom` helper in `Random.chpl` so this shouldn't be a big lift

Relevant numpy docs:
- https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.normal.html

### PR DESCRIPTION
**Pull request by stress-tess**
This PR (closes #3167) adds `normal` to our generators by transforming the results of standard normal. It also updates `standard_normal` to use generators to make sure we have the same multiple calls of the same function give different output as the other rand funcs. Adds goodness of fit testing against scipy to give confidence that we are actually drawing from the correct distribution

Added this to the docs with correct formatting of the math equations. I also added some deps to the conda env yaml files that were introduced in #3077. Also I changed the eol for that file in my pycharm from carriage return `\r\n` to just newline `\n`. This makes it look like every line in the yaml changed. In the `Files changed` tab you can click the setting wheel and hide whitespace to see what was actually added

### CODE REVIEW

**Comment by stress-tess** [comment]
converting to draft because I forgot to add documentation

File: pydoc/usage/random.rst [hunk 1]
```diff
@@ -17,10 +17,26 @@ To create a ``Generator`` use the ``default_rng`` constructor
 Features
 ==========
 
+choice
+---------
+.. autofunction:: arkouda.random.Generator.choice
+
```
Comment (ajpotts) [inline pydoc/usage/random.rst:23]: So why are some functions added to the usage section of the docs and not others?  What is the intended difference between the usage and api sections?

File: arkouda/random/_generator.py [hunk 2]
```diff
@@ -209,6 +209,55 @@ def integers(self, low, high=None, size=None, dtype=akint64, endpoint=False):
         self._state += size
         return create_pdarray(rep_msg)
 
+    def normal(self, loc=0.0, scale=1.0, size=None):
+        r"""
+        Draw samples from a normal (Gaussian) distribution
```
Comment (ajpotts) [inline arkouda/random/_generator.py:214]: The doc string for this one isn't rendering for me with `make doc`.

**Review by ajpotts — APPROVED** [review]
Looks good!  Just the two questions about the docs.

**Review by jaketrookman — APPROVED** [review]

File: pydoc/usage/random.rst (hunk 1 above)
Comment (stress-tess) [reply pydoc/usage/random.rst:23]: to be honest, this predates me. I don't really know. I think the api lists everything and usage is just meant for the most common stuff

File: arkouda/random/_generator.py (hunk 2 above)
Comment (stress-tess) [reply arkouda/random/_generator.py:214]: oh really?? that's odd. I might swing by to see what it's showing you

Comment (stress-tess) [reply arkouda/random/_generator.py:214]: yah I found the problem... just had a random `latex` keyword in the usage section `.. autofunction:: latex arkouda.random.Generator.normal`... should be fixed now!

**Pull request merged**